compiler/rustc_codegen_llvm/src/llvm/enzyme_ffi.rs

```diff
@@ -17,6 +17,11 @@ unsafe extern "C" {
     pub(crate) fn LLVMRustEraseInstFromParent(V: &Value);
     pub(crate) fn LLVMRustGetTerminator<'a>(B: &BasicBlock) -> &'a Value;
     pub(crate) fn LLVMRustVerifyFunction(V: &Value, action: LLVMRustVerifierFailureAction) -> Bool;
+    pub(crate) fn LLVMRustHasAttributeAtIndex(V: &Value, i: c_uint, Kind: AttributeKind) -> bool;
+
+    pub(crate) fn LLVMRustIsFunctionTy(Ty: &Type) -> bool;
+    pub(crate) fn LLVMRustIsArrayTy(Ty: &Type) -> bool;
```

Comment:
There are `TypeKind`s for these, can you use those instead?

Reply:
yes, thanks!